### PR TITLE
Add tweedledum to the list

### PR DESCRIPTION
**Pull request by boschmitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 **C++**
 - [QCOR](https://github.com/ORNL-QCI/qcor) - C++ language extension and associated compiler implementation for hybrid quantum-classical programming.
 - [ScaffCC](https://github.com/epiqc/ScaffCC) - Compilation, analysis and optimization framework for the Scaffold quantum programming language.
+- [tweedledum](https://github.com/boschmitt/tweedledum) - C++17 library for analysis, compilation/synthesis, and optimization of quantum circuits.
```
Comment (amirebrahimi) [inline README.md:198]: 😆 This is begging for a tweedledee library to be created now...

Comment (boschmitt) [reply README.md:198]: It [exists](https://github.com/boschmitt/tweedledee), but is kind of abandoned :(

It a parsing library reading quantum circuit in different formats (OpenQASM, Quil, .qc, ...). The overhead of maintaining both was too much though.

**Comment by markf94** [comment]
Thanks @boschmitt for the PR!

File: README.md (hunk 1 above)
Comment (markf94) [reply README.md:198]: who said that naming software packages can't be fun? ;)

**Pull request merged**